web/gui/dashboard_info.js

```diff
@@ -1235,6 +1240,40 @@ netdataDashboard.context = {
         info: 'Counters of file accesses. <code>Reference</code> is when there is a file access and the file is not present in the directory cache. <code>Miss</code> is when there is file access and the file is not found in the filesystem. <code>Slow</code> is when there is a file access and the file is present in the filesystem but not in the directory cache. Read more about <a href="https://www.kernel.org/doc/htmldocs/filesystems/the_directory_cache.html" target="_blank">directory cache</a>.'
     },
 
+    'md.health': {
+        info: 'Number of failed devices per MD array. '+
+        'Netdata retrieves this data from the <b>[n/m]</b> field of the md status line. '+
+        'It means that ideally the array would have <b>n</b> devices however, currently, <b>m</b> devices are in use. '+
+        '<code>failed disks</code> is <b>n-m</b>.'
+    },
+    'md.disks': {
+        info: 'Number of devices in use and in the down state. '+
+        'Netdata retrieves this data from the <b>[n/m]</b> field of the md status line. '+
+        'It means that ideally the array would have <b>n</b> devices however, currently, <b>m</b> devices are in use. '+
+        '<code>inuse</code> is <b>m</b>, <code>down</code> is <b>n-m</b>.'
+    },
+    'md.status': {
+        info: 'Completion progress of the ongoing operation.'
+    },
+    'md.expected_time_until_operation_finish': {
+        info: 'Estimated time to complete the ongoing operation. '+
+        'The time is only an approximation since the operation speed will vary according to other I/O demands.'
+    },
+    'md.operation_speed': {
+        info: 'Speed of the ongoing operation. '+
+        'Resyncing speed limits are specified in <code>/proc/sys/dev/raid/{speed_limit_min,speed_limit_max}</code> files. '+
```

Comment:
Please, consider to add space after comma `, speed_limit_max`.

Reply:
I've considered it and decided to not add it because it _looks_ better as it is (and I think (not sure tho) that this pattern is more common).